src/System.Private.CoreLib/shared/System/Reflection/AssemblyCompanyAttribute.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
Note that technically many of these attributes allow you to pass in null and then expose that from the associated property.  But I'm not aware of any situation in which that would be valuable, so I've left most of them as non-nullable.